README.md

```diff
@@ -121,7 +122,8 @@ $ sudo apt-get install android-tools-adb android-tools-fastboot autoconf bc \
 	bison cscope curl flex gdisk libc6:i386 libfdt-dev libftdi-dev \
 	libglib2.0-dev libhidapi-dev libncurses5-dev libpixman-1-dev \
 	libstdc++6:i386 libtool libz1:i386 mtools netcat python-crypto \
-	python-serial python-wand unzip uuid-dev xdg-utils xz-utils zlib1g-dev
+	python-serial python-wand unzip uuid-dev xdg-utils xz-utils zlib1g-dev \
+	xterm make libcap-dev libattr1-dev
```

Comment:
The list her has been sorted in alphabetic order to make it easy to see if something already is there. Please put the new dependencies in alphabetic order too.